addons/account/wizard/account_automatic_entry_wizard.py

```diff
@@ -145,7 +145,7 @@ def default_get(self, fields):
         res['move_line_ids'] = [(6, 0, move_line_ids.ids)]
 
         if any(move.state != 'posted' for move in move_line_ids.mapped('move_id')):
-            raise UserError(_('You can only change the period/account for posted journal items.'))
+            raise UserError(_("Oops! You can't change the period/account for reconciled items! Only the unreconciled ones are up for such an adventure!"))
```

Comment:
This is wrong. This error refers to the `posted` state. The one below is the one for unreconciled items.

Reply:
Better English
```suggestion
            raise UserError(_("Oops! You can't change the period/account for reconciled items! Only unreconciled ones are up for an adventure like that!"))
```